docs/community/topics/i18n.rst

```diff
@@ -125,26 +117,57 @@ Translate phrases, not words
 Full sentences and clauses must always be a single translatable string.
 Otherwise, you can get ``next page`` translated as ``suivant page`` instead of as ``page suivante``, etc.
 
-Deal with variables and markup in translations
+Dealing with variables and markup in translations
 `````````````````````````````````````````````````````````````
 
 If a variable (like the ``edit_page_provider_name`` theme option) is used as part of a phrase, it must be included within the translatable string.
-Otherwise, the word order in other languages can be incorrect.
-In a Jinja template, simply surround the translatable string with ``{% trans variable=variable %}`` and ``{% endtrans %}}`.
-For example: ``{% trans provider=provider %}Edit on {{ provider }}{% endtrans %}``
-The translatable string is extracted as the Python format string ``Edit on %(provider)s``.
-This is so that the same translatable string can be used in both Python code and Jinja templates.
-It is the translator's responsibility to use ``%(provider)s`` verbatim in the translation.
-
-If a non-translatable word or token (like HTML markup) is used as part of a phrase, it must also be included within the translatable string.
-For example: ``{% trans theme_version=theme_version|e %}Built with the <a href="https://pydata-sphinx-theme.readthedocs.io/en/stable/index.html">PyData Sphinx Theme</a> {{ theme_version }}.{% endtrans %}``
+Otherwise, the word order in other languages can be incorrect. In a Jinja template, simply surround the translatable string with ``{% trans variable=variable %}`` and ``{% endtrans %}}``. For example:
```

Comment:
```suggestion
Otherwise, the word order in other languages can be incorrect. In a Jinja template, surround the translatable string with ``{% trans variable=variable %}`` and ``{% endtrans %}``. For example:
```